pydocstringformatter/formatting/__init__.py

```diff
@@ -6,9 +6,11 @@
 from pydocstringformatter.formatting.formatter import (
     BeginningQuotesFormatter,
     ClosingQuotesFormatter,
+    FinalPeriodFormatter,
 )
 
 FORMATTERS: List[Formatter] = [
     BeginningQuotesFormatter(),
     ClosingQuotesFormatter(),
+    FinalPeriodFormatter(),
```

Comment:
```suggestion
        FinalPeriodFormatter(),
        ClosingQuotesFormatter(),
```
So we manage the line break if we go further than the alloted limit with the dot ?

Reply:
Hmm, but I depend on the number of line breaks to figure out what to do..

I think for now this is fine, until we have a good way of doing line wrapping? After which we'll probably need to update the order?

Reply:
Fair enough !